ui/admin/app/routes/account.js

```diff
@@ -15,9 +15,14 @@ export default class AccountRoute extends Route {
   // =methods
 
   /**
-   * If arriving here unauthenticated, redirect to index for further processing.
+   * If arriving here unauthenticated, redirect to index for further processing
+   * else redirect to change-password route since account route is empty.
    */
   beforeModel() {
-    if (!this.session.isAuthenticated) this.router.transitionTo('index');
+    if (!this.session.isAuthenticated) {
+      this.router.transitionTo('index');
+    } else {
+      this.router.transitionTo('account.change-password');
```

Comment:
I'm curious about the rationale for adding this else statement? If the user has arrived to this route unauthenticated, is the assumption that they must be locked out if the account route is empty and thus they should reset their pw?

Reply:
we don't display anything in the 'account' route (template is empty) so if the user were to go there it would be an empty screen so its best to redirect to the change-password route.